app/src/main/java/org/hwyl/sexytopo/control/io/basic/SketchJsonTranslater.java

```diff
@@ -161,6 +160,11 @@ public static PathDetail toPathDetail(JSONObject json) throws JSONException {
         }
 
         PathDetail pathDetail = new PathDetail(path, colour);
+
+        double epsilon = Space2DUtils.SketchEpsilon(pathDetail);
+        List<Coord2D> simplifiedPath = Space2DUtils.Simplify(path, epsilon);
+        pathDetail.setPath(simplifiedPath);
```

Comment:
I'm wondering whether we should set this behind a version guard clause. That is, only do this if the file was saved with a SexyTopo earlier than the current version. Although that would be a pain in the bum to do that the way the file is currently written. Guess this Douggie-Puker algo is idempotent? May well not be worth it.

Reply:
Assuming you use the same epsilon then yes, you'll get exactly the same result each time

Reply:
You probably understood what I meant, but just checking: we don't just need the result to be the same for the same input, but we also need the result to be the same for feeding the output of the first pass back into the algorithm. That is, we don't want the paths to get progressively simpler every time the file gets loaded.